packages/core/buidler.config.js

```diff
@@ -45,5 +185,13 @@ module.exports = {
   },
   mocha: {
     timeout: 1800000
+  },
+  etherscan: {
+    // The url for the Etherscan API you want to use.
+    // For example, here we're using the one for the Ropsten test network
+    url: process.env.ETHERSCAN_API_URL,
```

Comment:
Can't you detect the etherscan API URL dynamically depending on provider network? I think this would be a great feature as we can eliminate new env vars for user to keep